### PR TITLE
[12.0][MIG] sale_order_variant_mgmt

### PR DESCRIPTION
**Pull request by fuentes73**
Standard migration

### CODE REVIEW

File: sale_order_variant_mgmt/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#. Go to Sales > Sales > Quotations
```
Comment (ernestotejeda) [inline sale_order_variant_mgmt/readme/USAGE.rst]:
```suggestion
#. Go to Sales > Orders > Quotations
```

**Review by ernestotejeda — COMMENTED** [review]
:+1: LGTM

File: sale_order_variant_mgmt/readme/USAGE.rst (hunk 1 above)
Comment (fuentes73) [reply sale_order_variant_mgmt/readme/USAGE.rst]: Thanks, done

**Review by fuentes73 — COMMENTED** [review]
@Jortolsa @Abranes @ioans73 please review

**Review by ernestotejeda — APPROVED** [review]

**Review by pedrobaeza — COMMENTED** [review]
I have tried it and it's working, but we can improve it if we move the button for adding the variants at line level:

Selección_001

You can inspire from v13 where there's the perfect replacement for this module in https://github.com/odoo/odoo/tree/13.0/addons/sale_product_matrix (or maybe backport it).

**Comment by fuentes73** [comment]
Sure, no problem. Let me check.

File: sale_order_variant_mgmt/README.rst [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+* Tecnativa
+
+Contributors
+~~~~~~~~~~~~
+
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+  * Pedro M. Baeza
+  * Jairo Llopis
```
Comment (sergio-teruel) [inline sale_order_variant_mgmt/README.rst:94]: You can add yoursefl as a contributor

File: sale_order_variant_mgmt/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    'author': 'Tecnativa,'
+              'Odoo Community Association (OCA)',
+    'category': 'Sale',
+    'license': 'AGPL-3',
+    'website': 'https://github.com/OCA/product-variant',
+    'depends': [
+        'sale',
+        'web_widget_x2many_2d_matrix',
+    ],
+    'demo': [],
```
Comment (sergio-teruel) [inline sale_order_variant_mgmt/__manifest__.py]: Remove empty key

**Review by sergio-teruel — APPROVED** [review]
Ok form me

**Comment by pedrobaeza** [comment]
@fuentes010 have you been able to check what I suggested?

**Comment by fuentes73** [comment]
@pedrobaeza, yes and I've seen that [here](https://github.com/odoo/odoo/blob/12.0/addons/account/static/src/js/section_and_note_fields_backend.js) the widget SectionAndNoteListRenderer isn't returned. So can't be inherited.
There is a merged [PR](https://github.com/odoo/odoo/pull/34672) at Odoo but the target branch was moved to master.
Do you have any ideas or a possible workaround?

**Comment by pedrobaeza** [comment]
OK, then let's continue with this as it's fully functional and in 13.0 we will have the core one. Thanks for the migration

/ocabot merge

**Pull request merged**